### PR TITLE
Small readme cleanups

### PR DESCRIPTION
**Pull request by RecursiveForest**
I think the only controversial thing here is the removal of the "backwards incompatible changes" section. As we grow into our own application, not merely a fork of morituri, I don't think it's useful to keep a manually updated list of changes in the README. The changelog / git history should suffice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # Whipper
 
-[![license](https://img.shields.io/github/license/JoeLametta/whipper.svg)](https://github.com/JoeLametta/whipper/blob/master/LICENSE) [![Build Status](https://travis-ci.org/JoeLametta/whipper.svg?branch=master)](https://travis-ci.org/JoeLametta/whipper) [![GitHub (pre-)release](https://img.shields.io/github/release/joelametta/whipper/all.svg)](https://github.com/JoeLametta/whipper/releases/latest) [![IRC](https://img.shields.io/badge/irc-%23whipper%40freenode-brightgreen.svg)](https://webchat.freenode.net/?channels=%23whipper) [![GitHub Stars](https://img.shields.io/github/stars/JoeLametta/whipper.svg)](https://github.com/JoeLametta/whipper/stargazers) [![GitHub Issues](https://img.shields.io/github/issues/JoeLametta/whipper.svg)](https://github.com/JoeLametta/whipper/issues) [![GitHub contributors](https://img.shields.io/github/contributors/JoeLametta/whipper.svg)](https://github.com/JoeLametta/whipper/graphs/contributors)
+[![license](https://img.shields.io/github/license/JoeLametta/whipper.svg)](https://github.com/JoeLametta/whipper/blob/master/LICENSE)
```
Comment (RecursiveForest) [inline README.md]: these render the same but are easier to read/edit in plaintext broken up like this into separate lines.

File: README.md [hunk 2]
```diff
@@ -126,8 +130,6 @@ is correct, while
 
 is not, because the `-d` argument applies to the `cd` command.
 
-~~Check the man page (`whipper(1)`) for more information.~~ (currently not available as whipper's documentation is planned to be reworked ([Issue #73](https://github.com/JoeLametta/whipper/issues/73))).
```
Comment (RecursiveForest) [inline README.md]: We can add mention of a man page when we have one, but I don't think there's any need to call attention to the fact that it's a planned feature.

File: README.md [hunk 3]
```diff
@@ -299,12 +266,6 @@ And attach the gzipped log file to your bug report.
 
 Without `WHIPPER_LOGFILE` set, logging messages will go to stderr. `WHIPPER_DEBUG` accepts a string of the [default python logging levels](https://docs.python.org/2/library/logging.html#logging-levels).
 
-### Developing
-
-Pull requests are welcome.
-
-**WARNING:** As whipper is still under heavy development sometimes I will force push (`--force-with-lease`) to the non master branches.
-
 ## Credits
```
Comment (RecursiveForest) [inline README.md]: I sort of want to delete this section, as it's easy to see in the git history who the developers are, and we already have all these names (+ Freso's now) in the copyright section.

Comment (JoeLametta) [reply README.md]: :+1:

**Comment by MerlijnWajer** [comment]
I'd suggest to keep the ones in the README too. Many people don't get the git history when they're on their repo, but distros can often ship a README. Maybe the soon to be created manpage is another place for this?

**Comment by RecursiveForest** [comment]
I think the copyright/license file is the right place to keep information about copyright holders and developers; at the bottom of the man page is definitely a relevant place for it though, at least by standard.

I don't see many FOSS projects listing contributors in the README, which tends to be need-to-know-to-help-the-user information.

**Comment by MerlijnWajer** [comment]
It's particularly helpful if you want to find maintainers of software, though. And we should be doing this in each file (copyright header) too, with name and email addr.

**Comment by RecursiveForest** [comment]
I've restored the Credits section and resolved the conflicts with master.

**Comment by JoeLametta** [comment]
Merged, thanks.

**Pull request merged**